app/controllers/coronavirus_controller.rb

```diff
@@ -28,24 +69,8 @@ def update_draft
     else
       flash["alert"] = "Error received from GitHub - #{response.code}"
     end
-
-    redirect_to coronavirus_path
-  end
-
-  def publish
```

Comment:
The `publish` method has been renamed `publish_page`, and is called by two controller actions, `publish_landing` and `publish_business`